src/theme/components/button.ts

```diff
@@ -3,19 +3,82 @@ import { defineStyleConfig } from "@chakra-ui/react";
 export const buttonTheme = defineStyleConfig({
   baseStyle: {
     backgroundColor: "primary.600",
+    borderRadius: "base",
+    // token?
     color: "white",
     px: 6,
     py: 3,
   },
   sizes: {
-    sm: {},
-    md: {},
-    lg: {},
+    sm: {
+      height: 11,
+    },
+    md: {
+      height: 12,
+    },
+    lg: {
+      height: 14,
+    },
   },
   variants: {
-    primary: {},
-    secondary: {},
-    tertiary: {},
+    primary: {
+      // token?
+      boxShadow: "0 1.5 1.5 0 rgba(35, 78, 82, 0.08)",
+      _disabled: {
+        backgroundColor: "primary.500",
+        color: "white",
+        // token?
```

Comment:
`opacity` is one of the few properties that doesn't have tokens so you're good here. 
![image](https://github.com/NYCPlanning/ae-design-system/assets/9055367/a22d240b-afb6-4962-b2c1-1dc65266e378)

(from https://chakra-ui.com/docs/styled-system/style-props#color-and-background-color)